bevy_lint/src/lints/insert_event_resource.rs

```diff
@@ -49,7 +49,10 @@ use std::borrow::Cow;
 
 declare_bevy_lint! {
     pub INSERT_EVENT_RESOURCE,
-    SUSPICIOUS,
+    // The only time this lint would be a false-positive is within
```

Comment:
This can also be used for manually opting out of Bevy's event clearing strategy. You probably don't need to ever do that these days, but it's not a totally useless thing to do.

Reply:
Ah, that makes sense! I'll revert this back to suspicious.